code/datums/mutations/body.dm

```diff
@@ -137,7 +137,7 @@
 			if(1)
 				owner.emote("twitch")
 			if(2 to 3)
-				owner.say("[prob(50) ? ";" : ""][pick("SHIT", "PISS", "FUCK", "CUNT", "COCKSUCKER", "MOTHERFUCKER", "TITS")]", forced="tourette's syndrome")
+				owner.say("[prob(50) ? ";" : ""][pick("GÓWNO", "SIKI", "KURWA", "PEDAŁ", "LACHOCIĄG", "SKURWYSYN", "CYCKI")]", forced="tourette's syndrome")
```

Comment:
wiem że to nie jest bezpośrednie tłumaczenie, ale zamiast tego "siki" dałbym tam "chuj". wydaje mi sie że troszke bardziej pasuje